src/command.rs

```diff
@@ -434,6 +434,60 @@ pub async fn logout(registry: RegistryUri) -> miette::Result<()> {
     credentials.write().await
 }
 
+use axum::{
+    extract,
+    http::{header, StatusCode},
+    response::IntoResponse,
+    routing::get,
+    Router,
+};
+use bytes::Bytes;
+use std::collections::HashMap;
+use std::sync::{Arc, RwLock};
+
+type State = Arc<RwLock<HashMap<String, Bytes>>>;
+
+/// Run a minimal registry for local testing
+pub async fn test_registry(listen: SocketAddr) -> miette::Result<()> {
+    let state = Arc::new(RwLock::new(HashMap::<String, Bytes>::new()));
```

Comment:
All of this should be wrapped in:
```
#[cfg(test)]
mod test_registry {
   ...
}
```

As it shouldn't be compiled when releasing actual binary.

Reply:
I am not quite sure. I intended `test-registry` for users who are following a tutorial or want to test the publish/add commands without actually publishing packages to a a real registry.

If @mara-schulke and you think it's better not to add such a command, i can remove it entirely, and move the `test_registry` function to `test/`.

Reply:
Yes, there should not be such a command for the user. 

I am questioning the usefulness of this registry tho! You can't properly use buffrs anyway until you have a artifactory instance and for e2e testing we use something different.

Reply:
Citing Mara:
> Yes, there should not be such a command for the user.

plus

> Okay, so after re-reading everything, I think you are indeed adding a lot of good tests here. Feel free to keep this here, even tho I think it should be replaced by the proper registry longterm.

I think the consensus is to keep everything, but put it behind `#[cfg(test)]`.

Reply:
> You can't properly use buffrs anyway until you have a artifactory instance

That's why I made this command. I wanted to be able to follow along the tutorial without having to upload packages to the artifactory.

I'll just remove the command.

Reply:
Unless I'm mistaken, all of the test code is still compiled unconditionally (including release binary).

I meant not only the `test_registry`, but basically all of the code added as part of this PR.

Reply:
There is no more change to `src/`. All the code is now in `tests/`.

Reply:
True, I was looking at `Cargo.toml`, but missed that the deps now live in `dev-dependencies`. Resolving.